<?xml version="1.0"?>
<launch>

  <!-- Navigation Single Robot  -->
  <include file="$(find turtlebot2i_navigation)/launch/turtlebot2i_navigation_single.launch">
    <arg name="cmd_vel_mux" value="true"/>
  </include>

  <!-- KeyOp  -->
  <include file="$(find turtlebot2i_navigation)/launch/turtlebot2i_keyop.launch">
    <arg name="cmd_vel_mux" value="true"/>
  </include>

  <!-- Cmd_vel_mux  -->
  <include file="$(find turtlebot2i_navigation)/launch/turtlebot2i_cmd_vel_mux.launch"/>

  <!-- Scene Graph Generator & Parser -->
  <include file="$(find turtlebot2i_safety)/launch/labeling/turtlebot2i_scene_graph_labeling.launch"/>

</launch>
